deployment/hasura/metadata/databases/AerieMerlin/tables/public_plan_snapshot_activities.yaml

```diff
@@ -16,3 +16,26 @@ array_relationships:
         column_mapping:
           id: directive_id
           snapshot_id: snapshot_id
+select_permissions:
+  - role: aerie_admin
+    permission:
+      columns: [snapshot_id, id, name, source_scheduling_goal_id, created_at, last_modified_at, start_offset, type,
+                arguments, last_modified_arguments_at, metadata, anchor_id, anchored_to_start]
```

Comment:
Make sure to add `last_modified_by` here once https://github.com/NASA-AMMOS/aerie/pull/1041 is merged 👍

Reply:
Done!